molecule/cookiecutter/driver/docker/{{cookiecutter.repo_name}}/molecule.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
> I don't get how the variable in the path name works for this file?

It's a cookiecutter thing.  Creates a directory named what we call the repo.  In our case the repo is the name of the role we are creating.